pages/users/[id].tsx

```diff
@@ -108,10 +108,15 @@ export default function User({ loginContext }: Props) {
 
         if (
           'error' in user ||
+          API.isGenericError(user as API.GenericApiError) ||
           'error' in events ||
+          API.isGenericError(events as API.GenericApiError) ||
           'error' in allTimeMetrics ||
+          API.isGenericError(allTimeMetrics as API.GenericApiError) ||
           'error' in weeklyMetrics ||
-          'error' in metricsConfig
+          API.isGenericError(weeklyMetrics as API.GenericApiError) ||
+          'error' in metricsConfig ||
+          API.isGenericError(metricsConfig as API.GenericApiError)
```

Comment:
This construct isn't my favorite I've ever written -- if there's a common and easier approach I'm happy to try it -- I spend a while messing around with `ApiError` but found it kept complaining.